src/coreclr/jit/gentree.cpp

```diff
@@ -16201,7 +16201,9 @@ GenTree* Compiler::gtNewRefCOMfield(GenTree*                objPtr,
             // helper needs pointer to struct, not struct itself
             if (pFieldInfo->helper == CORINFO_HELP_SETFIELDSTRUCT)
             {
-                assg = impGetStructAddr(assg, CHECK_SPILL_ALL, true);
+                // TODO: deal with flags
```

Comment:
what does "deal with flags" mean?

Reply:
The code should be audited whether flags matter or not, and if yes we should probably read the address with them and use a temp and its address.